shuup/core/api/category.py

```diff
@@ -38,6 +43,8 @@ class CategoryFilter(FilterSet):
     shop = django_filters.ModelChoiceFilter(name="shops",
                                             queryset=Shop.objects.all(),
                                             lookup_expr="exact")
+    slug = django_filters.CharFilter(name="translations__slug",
```

Comment:
I didn't filter by `identifier` because user can't see it on the Shuup Admin. I can add a field to filter by root category.